spec/aws_loader_spec.rb

```diff
@@ -1,25 +1,53 @@
+# Dummy class for testing module
+class ModuleTest
+  include AwsLoader
+end
+
 RSpec.describe AwsLoader do
 
   let (:mod_test) { ModuleTest.new }
 
   context 'aws.yml file' do    
     context '#configure_aws_file' do
-      it 'should return false for aws.yml file check and call #configure_aws_file and create a new .yml file' do
+      it 'should call #configure_aws_file' do
         stub_const('AwsLoader::AWS_PATH', nil)
-        allow(File).to receive(:exists?).and_return(false)
-        expect(STDOUT).to receive(:puts).and_return('No `aws.yml` file present!')
-        expect(STDOUT).to receive(:puts).and_return('Created file and set the appropriate values!')
+        allow(File).to receive(:exists?).with(AwsLoader::AWS_PATH).and_return(false)
         expect(mod_test).to receive(:generate_aws_yml_file)
         mod_test.configure_aws_file
```

Comment:
Replace this line with expect{ mod_test.configure_aws_file }.to raise_error(SystemExit).
Uncomment this test and run.

Reply:
Run the spec for `aws_loader_spec.rb` - check the number of dots versus the number.
Then add the line as given in the comment and run the specs again.
Rspec goes into a funk when there is an `exit` statement in the code - so it is better that you handle it.